CONTRIBUTING.md

```diff
@@ -407,10 +403,8 @@ git push origin
 git switch -c release/<RELEASE_VERSION>
 git add CHANGELOG.md (see Update Changelog file section)
 Update manually `chutneyVersion`  in [idea-plugin/gradle.properties](https://github.com/chutney-testing/chutney/blob/main/idea-plugin/gradle.properties)
-Update manually `chutneyVersion`  in [kotlin-dsl/gradle.properties](https://github.com/chutney-testing/chutney/blob/main/kotlin-dsl/gradle.properties)
-mvn versions:set -DnewVersion=<RELEASE_VERSION> -DgenerateBackupPoms=false -f chutney/pom.xml
-mvn versions:set-scm-tag -DnewTag=<RELEASE_VERSION> -DgenerateBackupPoms=false -f chutney/pom.xml
-mvn versions:set -DnewVersion=<RELEASE_VERSION> -DgenerateBackupPoms=false -f example/pom.xml
+mvn versions:set -DnewVersion=<RELEASE_VERSION> -DgenerateBackupPoms=false
```

Comment:
This doesn't work for me when i tried with valid version number

Reply:
You have to run it from parent folder and not from /chutney folder